tools/ci/cirrus_arm.yml

```diff
@@ -97,10 +94,6 @@ macos_arm64_test_task:
     RUNNER_OS="macOS"
     SDKROOT=/Applications/Xcode-14.0.0.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk
     
-    # NOTE: OpenBLAS is not used in this job; if that's done in the future, ensure
-    # PKG_CONFIG_PATH points to the directory containing the openblas.pc file
-    # that's installed with the cibw_before_build.sh command.
-    # used for installing OpenBLAS/gfortran
```

Comment:
Wheel builds weren't changed, so I think the changes in this file should be reverted?